library/ccm.c

```diff
@@ -49,6 +49,13 @@
 #endif /* MBEDTLS_PLATFORM_C */
 #endif /* MBEDTLS_SELF_TEST && MBEDTLS_AES_C */
 
+#if defined(MBEDTLS_CCM_ALT)
+#define MBEDTLS_CCM_DEFAULT_INLINE
+#else
+#define MBEDTLS_CCM_DEFAULT_INLINE static inline
+#endif
```

Comment:
Please note that in the past we have used something like this for `inline`:
```
#if ( defined(__ARMCC_VERSION) || defined(_MSC_VER) ) && \
    !defined(inline) && !defined(__cplusplus)
#define inline __inline
#endif
```
Please refer to `include/mbedtls/ssl_internal.h` for an example.

Reply:
Thanks for your comment, but the purpose for this MACRO is to define whether the functions are inline or not, not to define what is inline. I'll add your suggestion for defining `inline` though